pkg/cli/admin/release/info.go

```diff
@@ -735,16 +740,47 @@ func (i *ReleaseInfo) Platform() string {
 }
 
 func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*ReleaseInfo, error) {
+	opts := extract.NewOptions(genericclioptions.IOStreams{Out: o.Out, ErrOut: o.ErrOut})
+	opts.SecurityOptions = o.SecurityOptions
+	opts.ImageSourceOptions = o.ImageSourceOptions
+	opts.FileDir = o.FileDir
+
+	setPrefix, imageSources, err := imagemanifest.GetImageSourcePrefixes(opts.ImageSourceOptions.ImageContentSourcePolicyList, image)
```

Comment:
I'd suggest either calling this `useMirrors` or check `imageSources` length and act accordingly in the condition below.

Reply:
moved `setPrefix` to `useImageContentSources`